python/trifinger_simulation/collision_objects.py

```diff
@@ -106,42 +103,60 @@ def __init__(
             lateralFriction=lateral_friction,
             spinningFriction=spinning_friction,
             restitution=restitution,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
         )
 
     def set_state(self, position, orientation):
         """
-        Resets the block state to the provided position and
-        orientation
+        Resets the cuboid to the provided position and orientation
 
         Args:
-            position: the position to which the block is to be
-                set
-            orientation: desired to be set
+            position: New position.
+            orientation: New orientation.
         """
         pybullet.resetBasePositionAndOrientation(
             self.block,
             position,
             orientation,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
         )
 
     def get_state(self):
         """
         Returns:
-            Current position and orientation of the block.
+            Current position and orientation of the cuboid.
         """
         position, orientation = pybullet.getBasePositionAndOrientation(
             self.block,
-            **self._kwargs,
+            physicsClientId=self._pybullet_client_id,
         )
         return list(position), list(orientation)
 
     def __del__(self):
         """
-        Removes the block from the environment
+        Removes the cuboid from the environment.
         """
         # At this point it may be that pybullet was already shut down. To avoid
         # an error, only remove the object if the simulation is still running.
-        if pybullet.isConnected(**self._kwargs):
-            pybullet.removeBody(self.block, **self._kwargs)
+        if pybullet.isConnected(self._pybullet_client_id):
+            pybullet.removeBody(self.block, self._pybullet_client_id)
+
```

Comment:
Just a small thing I remembered: maybe it is not in the scope of this pr, but we could finally add a method for getting the linear and angular velocities of the cuboid (getBaseVelocity) so in case people do have a way of getting reasonably accurate estimate of the object velocity from the tracker, there is an analogue in simulation too for it.